specs/booking.yml

```diff
@@ -117,8 +117,8 @@ tags:
       | **RESERVED** | Booking created by calling [Create booking](#operation/bookingCreate) endpoint. |
       | **CONFIRMED** | Booking confirmed, e.g. TAXI driver have accepted this order. We are waiting for this state update from TSP by periodically calling [Get booking](#operation/bookingGet) or waiting for webhook invocation via [Update booking state](#operation/backendBookingReceive) endpoint invocation. |
       | **ARRIVED** | Not applicable for all modes of transportation. For TAXI means driver have arrived to the pickup point. We are waiting for this state update from TSP. |
-      | **ACTIVATED** | Booking has started. For TAXI means TAXI has picked up the passenger. We are waiting for this state update from TSP |
-      | **ON_HOLD** | Booking still active but have been paused, for example for Car Sharing - lock the car but not finish the ride. We are waiting for this state update from TSP. | 
+      | **ACTIVATED** | Booking has started. For TAXI means TAXI has picked up the passenger. We are waiting for this state update from TSP for Taxi and will send this state update via [Update booking](#operation/bookingUpdate) call for car sharing and micromobility providers (to unlock car or a scooter and start a ride). |
```

Comment:
> We are waiting for this state update from TSP for Taxi
Not just for only taxi, there are a few different providers that also provider ACTIVATED state

Reply:
I would really like to be specific on modes of transportation for this case. Besides TAXI I can think of PT too, anything else?

Reply:
I can't remember from top of my head